camera.py

```diff
@@ -0,0 +1,57 @@
+import requests
+from requests.auth import HTTPDigestAuth
+
+class camera:
+    def __init__(self, ID="", url="", manufacturer="", calendar="", pos=-1, status=0):
+        self.ID = ID
+        self.url = url
+        self.manufacturer = manufacturer
+        self.calendar = calendar
+        self.pos = pos
+        self.status = status
+
+        print("Initialized: ", self)
+    
+    def __str__(self):
+        return f"\'{self.ID}\' @ \'{self.url}\' (Type: \'{self.manufacturer}\') (Current Position: {self.pos})"
+    
+    def updateCalendar(self, calendar):
+        self.calendar = calendar
```

Comment:
Python isn't Java 😁
It would be totally acceptable to do something like:

```py
cam = Camera(…)
cam.calendar = new_calendar
```

No need to have a setter for everything.